plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClient.java

```diff
@@ -130,7 +132,7 @@
 
     private static final int PRECISION_OF_UNSPECIFIED_NUMBER = 127;
 
-    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("uuuu-MM-dd");
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("uuuu-MM-dd HH:mm:ss");
```

Comment:
Revert. The date formatter should not render hours/minutes/seconds.

Instead use `TO_DATE(?, 'SYYYY-MM-DD')`